bin/opencanaryd

```diff
@@ -47,23 +37,18 @@ elif [ "${cmd}" == "--usermodule" ]; then
 elif [ "${cmd}" == "--restart" ]; then
     pid=`sudo cat "${PIDFILE}"`
     sudo kill "$pid"
-    if [ ! -f ~/.opencanary.conf -a ! -f opencanary.conf -a ! -f /etc/opencanaryd/opencanary.conf ]; then
-        echo "[e] No config file found, please create one with \"opencanaryd --copyconfig\""
-        exit 1
-    fi
     sudo "${DIR}/twistd" -y "${DIR}/opencanary.tac" --pidfile "${PIDFILE}" --syslog --prefix=opencanaryd
-
 elif [ "${cmd}" == "--stop" ]; then
     pid=`sudo cat "${PIDFILE}"`
     sudo kill "$pid"
 elif [ "${cmd}" == "--copyconfig" ]; then
-    if [ -f ~/.opencanary.conf ]; then
-        echo "[e] A config file already exists at ${HOME}/.opencanary.conf, please move it first"
+    if [ -f /etc/opencanaryd/opencanary.conf ]; then
+        echo "A config file already exists at /etc/opencanaryd/opencanary.conf, please move it first"
         exit 1
     fi
     defaultconf=$(python -c "from __future__ import print_function; from pkg_resources import resource_filename; print(resource_filename('opencanary', 'data/settings.json'))")
-    cp "${defaultconf}" ~/.opencanary.conf
-    echo -e "[*] A sample config file is ready (${HOME}/.opencanary.conf)\n"
+    sudo cp "${defaultconf}" /etc/opencanaryd/opencanary.conf
```

Comment:
So there is a failure here if there is no `/etc/opencanaryd/` directory. Can we add a check for that or just try create a new one and fail silently if it exists already

Reply:
:facepalm: I was thinking about exactly that when I wrote it but then forgot to add the mkdir bit in... I'll update the PR.